itest/lnd_multi-hop_force_close_test.go

```diff
@@ -45,6 +50,18 @@ var multiHopForceCloseTestCases = []*lntest.TestCase{
 		Name:     "multihop receiver preimage claim leased",
 		TestFunc: testMultiHopReceiverPreimageClaimLeased,
 	},
+	{
```

Comment:
do you think it makes sense to readd the static_remote type ?

Reply:
na think we should deprecate legacy channels - also remember we removed the test for it in a previous PR?